src/http_server/api/v1/metrics.c

```diff
@@ -211,6 +246,17 @@ void cb_metrics_prometheus(mk_request_t *request, void *data)
         buf->users--;
         return;
     }
+    sds_head = FLB_SDS_HEADER(sds);
```

Comment:
Calling `FLB_SDS_HEADER` should not be necessary. In general, the SDS header is an internal implementation detail of the library that callers do not need to interact with.

Reply:
Never mind, I see that this is to get the length of the SDS, since it seems that the `flb_sds_len` function in `flb_sds.h` is marked as static.

@edsiper Why is that? Can we expose the `flb_sds_len` function and export it? That seems better than requiring caller code to get the header and access the len field directly.